http/server_test.ts

```diff
@@ -973,14 +973,14 @@ Deno.test(
     // the expected backoff delay.
     for (let i = 0; i < rejectionCount; i++) {
       assertEquals(
-        listener.acceptCallIntervals[i] >= expectedBackoffDelays[i],
+        listener.acceptCallIntervals[i]! >= expectedBackoffDelays[i]!,
```

Comment:
Can you please justify these non-null assertions?

Reply:
`rejectionCount` is the length of `expectedBackoffDelays` and both are not modified during the execution. The length of `listener.acceptCallIntervals` is `rejectionCount` by the initialization